### PR TITLE
fix preview doesn't work (#23)

### LINKED ISSUE #23: Live preview doesn't work on mac yet
_Originally posted by @alse in https://github.com/Envek/obs-studio-node-example/pull/19#issuecomment-679145164_

### PR DESCRIPTION
**Pull request by DavidSongzw**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,7 +47,11 @@ yarn install
 
 Start the app from a native terminal, not an integrated terminal like VSCode's.
 
-There is some known issues with MacOS: preview doesn't work ([#23](https://github.com/Envek/obs-studio-node-example/issues/23)), obs64 process doesn't shutdown on app quit ([#22](https://github.com/Envek/obs-studio-node-example/issues/22)), errors in log files ([#18](https://github.com/Envek/obs-studio-node-example/issues/18)). If you can help with those, please fix them and open a pull request!
+`node-window-rendering` support MacOS to preview
+
+There is some known issues with MacOS: ~~preview doesn't work ([#23](https://github.com/Envek/obs-studio-node-example/issues/23))~~
+
+obs64 process doesn't shutdown on app quit ([#22](https://github.com/Envek/obs-studio-node-example/issues/22)), errors in log files ([#18](https://github.com/Envek/obs-studio-node-example/issues/18)). If you can help with those, please fix them and open a pull request!
```
Comment (Envek) [inline README.md:54]: You can just drop link to #23 from the list.
```suggestion
There is some known issues with MacOS: obs64 process doesn't shutdown on app quit ([#22](https://github.com/Envek/obs-studio-node-example/issues/22)), errors in log files ([#18](https://github.com/Envek/obs-studio-node-example/issues/18)). If you can help with those, please fix them and open a pull request!
```

**Review by Envek — COMMENTED** [review]
Wow! Cool! Thank you very much!

@alse, @psd-coder, could someone check this PR please?

**Comment by alse** [comment]
Thanks @DavidSongzw! @Envek the preview works for me

**Comment by psd-coder** [comment]
Preview works for me too. But seems recording isn't. After recording videos folder is empty.

**Comment by alse** [comment]
Recording works for me

**Comment by chfeizy** [comment]
Preview and recording is success,
but the camera can't work

**Comment by Envek** [comment]
Thank you folks for testing it!

@DavidSongzw, thank you very much for the fix!

**Pull request merged**